src/main/java/be/syntra/devshop/DevshopBack/entities/User.java

```diff
@@ -46,11 +45,11 @@
     @Column(name = "archived_carts")
     @ManyToMany(cascade = CascadeType.ALL)
     @JoinTable(
-            name = "USER_CART",
+            name = "USER_SHOPORDER",
             joinColumns = {@JoinColumn(name = "user_id", referencedColumnName = "user_id")},
-            inverseJoinColumns = {@JoinColumn(name = "cart_id", referencedColumnName = "cart_id")},
-            foreignKey = @ForeignKey(name = "cart_fk"))
-    private List<Cart> archivedCarts;
+            inverseJoinColumns = {@JoinColumn(name = "shoporder_id", referencedColumnName = "shoporder_id")},
+            foreignKey = @ForeignKey(name = "shoporder_fk"))
+    private List<ShopOrder> archivedShopOrders;
```

Comment:
I don't think this is the correct name for this List. As of this moment, a user has indeed only archived ShopOrders. But there's already a boolean in place to implement ShopOrders that aren't finalized yet. I suggest 'shopOrders'?

Reply:
indeed, fixed